.gitlab-ci-gputest.yml

```diff
@@ -48,24 +48,24 @@ test:s9300:
       --parallel $PARALLEL_JOBS
   allow_failure: true
 
-test:mi25:
-  extends: .test:cmake-latest
-  needs:
-    - build:cmake-latest
-  tags:
-    - mi25
-    - rocm
-  variables:
-    PARALLEL_JOBS: 2
-    TEST_GFX_IPS_REGEX: gfx900
-  script:
-    - cd $BUILD_LATEST_DIR
-    - $SUDO_CMD $CMAKE_LATEST_PATH/bin/cmake
-      -D CMAKE_PREFIX_PATH=/opt/rocm
-      -P $CI_PROJECT_DIR/cmake/GenerateResourceSpec.cmake
-    - $SUDO_CMD $CMAKE_LATEST_PATH/bin/ctest
-      --output-on-failure
-      --repeat-until-fail 2
-      --tests-regex "hip|$TEST_GFX_IPS_REGEX"
-      --resource-spec-file ./resources.json
-      --parallel $PARALLEL_JOBS
+#test:mi25:
+#  extends: .test:cmake-latest
+#  needs:
+#    - build:cmake-latest
+#  tags:
+#    - mi25
+#    - rocm
+#  variables:
+#    PARALLEL_JOBS: 2
+#    TEST_GFX_IPS_REGEX: gfx900
+#  script:
+#    - cd $BUILD_LATEST_DIR
+#    - $SUDO_CMD $CMAKE_LATEST_PATH/bin/cmake
+#      -D CMAKE_PREFIX_PATH=/opt/rocm
+#      -P $CI_PROJECT_DIR/cmake/GenerateResourceSpec.cmake
+#    - $SUDO_CMD $CMAKE_LATEST_PATH/bin/ctest
+#      --output-on-failure
+#      --repeat-until-fail 2
+#      --tests-regex "hip|$TEST_GFX_IPS_REGEX"
+#      --resource-spec-file ./resources.json
+#      --parallel $PARALLEL_JOBS
```

Comment:
Prefer to delete code instead of leaving it commented out.

Reply:
Deleted